elrond-wasm-derive/src/generate/only_owner_gen.rs

```diff
@@ -0,0 +1,16 @@
+use crate::model::{Method, PublicRole};
+
+pub fn generate_only_owner_snippet(m: &Method) -> proc_macro2::TokenStream {
+	if let PublicRole::Endpoint(endpoint_metadata) = &m.public_role {
+		if endpoint_metadata.only_owner {
+			let message = format!(
+				"Only owner can call {} !",
```

Comment:
The "!" may seem too verbose. Please consider removing the "!" sign

Reply:
There is no need to have the endpoint name in the error message, as you already know the endpoint you called. Would be more efficient if all had the same error message, as that would just have a single string message constant in the output code instead of one message for each owner-only endpoint.